charmcraft.yaml

```diff
@@ -56,12 +60,48 @@ config:
       default: 6
       description: |
         The interval in hours between each scheduled image builds.
-    openstack-clouds-yaml:
+    os-auth-url:
```

Comment:
`os` is a bit confusing since it is so commonly refers to operating system, I would be ok with `openstack-*` it is a bit longer and quite clear

Reply:
sounds good, thank you!